src/stdlib_stats_distribution_PRNG.fypp

```diff
@@ -0,0 +1,151 @@
+#:include "common.fypp"
+module stdlib_stats_distribution_PRNG
+    use stdlib_kinds, only: int8, int16, int32, int64
+    use stdlib_error
+    implicit none
+    private
+    integer, parameter :: MAX_INT_BIT_SIZE = bit_size(1_int64)
+    integer(int64), save :: st(4)     ! internal states for xoshiro256ss function
+    integer(int64), save :: si = 614872703977525537_int64 ! default seed value
+    logical, save :: seed_initialized = .false.
```

Comment:
`save` attribute is redundant for module variables. I suggest to remove it. Or do you prefer to keep it because of the intent it communicates? To me it's more confusing than not.

```suggestion
    integer(int64) :: st(4) ! internal states for xoshiro256ss function
￼   integer(int64) :: si = 614872703977525537_int64 ! default seed value
￼   logical :: seed_initialized = .false.
```